src/main/scala/group/scala/karazin/mongo4s/model.scala

```diff
@@ -192,10 +192,11 @@ object model:
 
     object CollStats:
       final case class LatencyStats(histograms: Boolean)
+      final case class StorageStats(scale: Int)
 
       // Should be rewritten with a builder
       final case class Command(latencyStats: Option[LatencyStats],
-                               storageStats: Option[EmptyObject],
+                               storageStats: Option[EmptyObject | StorageStats],
```

Comment:
Could storage stats be `EmptyObject`?

Reply:
according to the docs:
- Specify an empty document (i.e. storageStats: {}) 
- Specify the scale factor (i.e. storageStats: { scale: <number> })

Reply:
But do we need `Option` then?

Reply:
In the same docs:
The $collStats stage accepts an argument document with the following **optional** fields:...(includes `storageStats`)
Which is kind of confusing

Reply:
Cool, thanks.